### ISSUE TITLE
ForgeWrapper does not run forge >=36.1.66

### ISSUE THREAD

**Issue by deeg05** [ZekerZhayard/ForgeWrapper#6]
```
Some extra libraries are missing! Run the installer to generate them now.
Failed to start Minecraft:
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
Caused by: java.lang.reflect.InvocationTargetException
Exiting with -1
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:44)
	... 8 more
Caused by: java.lang.NoSuchMethodError: net.minecraftforge.installer.json.Util.loadInstallProfile()Lnet/minecraftforge/installer/json/Install;
	at io.github.zekerzhayard.forgewrapper.installer.Installer.install(Installer.java:12)
	... 13 more
Process exited with code 255.
```

**Comment by papycochon** [comment]
le deeg05

**Comment by ZekerZhayard** [comment]
Fixed in 1.5.1
